### DISCUSSION TITLE
Change « splash screen » of lbb

### DISCUSSION THREAD

**Discussion by asuatton** [outdoorbits/little-backup-box#200] [General]
Do you know how to change the picture on the start program ? If it’s possible ?   
thanks

**Comment by outdoorbits** [comment]
You want to use your own background-images for the web UI. right? The 
images you can select in config in 
"/var/www/little-backup-box/img/backgrounds/". In you put your own 
images here, you can select them in config.

**Reply by asuatton** [reply]
No I mean the splash screen write « Little backup box » that come when the program start. (Like a boot screen but comes when program start )

**Comment by outdoorbits** [comment]
O.k., just replace /var/www/little-backup-box/little-backup-box.bmp (monochrome displays) and/or  /var/www/little-backup-box/little-backup-box.jpg (RGB displays).
If you can provide better images for this project, welcome!

**Reply by Stefankante** [reply]
thanks for the trick!!!
![IMG_3776 copy](https://user-images.githubusercontent.com/35141284/230231236-711dcd7f-0495-4fb7-95b6-4911895e3e43.jpg)

**Reply by keeratsawhney** [reply]
> thanks for the trick!!!
> ![IMG_3776 copy](https://user-images.githubusercontent.com/35141284/230231236-711dcd7f-0495-4fb7-95b6-4911895e3e43.jpg)
> 

What is this setup ? 😵 Is this Your LBB on top of Powerbank ?

**Comment by keeratsawhney** [comment]
@outdoorbits ![20241209_130559.jpg](https://github.com/user-attachments/assets/2242652f-9b1b-40d7-8845-451fc9f87d5f)

Help solving Permissions.

**Comment by outdoorbits** [comment]
You have replace the file as root from ssh. Do not set a root password because this would break lbb.

**Reply by keeratsawhney** [reply]
Should we leave the password field blank at the time of installation of the os ? Or How not to set the password? I didn't modified anything in the settings, At the time of installation I gues yes.

**Comment by outdoorbits** [comment]
No, it is just about the root password. If you might have the idea to login as root at ftp this would fail.
Just upload the file to /media by ftp then use ssh and move the file by command
sudo mv /media/... /var/www...

**Reply by keeratsawhney** [reply]
> No, it is just about the root password. If you might have the idea to login as root at ftp this would fail.
> Just upload the file to /media by ftp then use ssh and move the file by command
> sudo mv /media/... /var/www...

When ever I am using Sudo as a command it asks me for password, Unfortunately it doesn't accept any password. Can you help me is there any default password?

**Comment by outdoorbits** [comment]
There is none set by LBB. Looks like you have to install new ...

**Reply by keeratsawhney** [reply]
If I take a backup of LBB settings and upload it back, I hope it will bring back all LBB settings, And Passwords are not stored back or they are also restored with it ?

**Reply by keeratsawhney** [reply]
While making Os Installation, I set the Username to Pi and The Password. is that supposed to be root password?

**Comment by outdoorbits** [comment]
Everything related to LBB will be stored, even the password of the web UI user lbb. The root password is not part of that and isn't stored. 
But I don't understand where your root password comes from. Did you set it?

**Comment by outdoorbits** [comment]
Not at all

**Reply by keeratsawhney** [reply]
Tried to reinstall, Still it shows two user Pi and Lbb-desktop. and there is a sudo password set for Lbb-desktop. I am not sure of this ![20241211_030219.jpg](https://github.com/user-attachments/assets/163cfc69-0907-4c67-b39e-7a555f003cd0)

@outdoorbits can you please look into this

**Comment by keeratsawhney** [comment]
> No, it is just about the root password. If you might have the idea to login as root at ftp this would fail.
> Just upload the file to /media by ftp then use ssh and move the file by command
> sudo mv /media/... /var/www...

Can you guide me how to setup FTP ? I cannot see any details or tabs to configure ftp. Help would be appreciated

**Reply by outdoorbits** [reply]
There is nothing to setup. Just use a ftp client, ftp://your.ip. Now you have access to the file system. But users pi or lbb are not allowed to write in /var/www. That's why you have to upload your file to /media, login via ssh and move the file in place: 
sudo mv /media/your_file /var/www...
